debezium-core/src/main/java/io/debezium/pipeline/spi/OffsetContext.java

```diff
@@ -69,4 +70,6 @@
      * Records the name of the collection and the timestamp of the last event
      */
     void event(DataCollectionId collectionId, Instant timestamp);
+
+    TransactionContext getTransactionContext();
```

Comment:
Please add a comment, as it's a public contract and it'll be beneficial for us, too.

Reply:
Added